admin/app/templates/views/two-factor-email.html

```diff
@@ -14,7 +14,7 @@ <h1 class="heading-large">{{ title }}</h1>
     <p>Clicking the link will open Notify in a new browser window, so you can close this one.</p>
     {{ page_footer(
       secondary_link=url_for('main.email_not_received'),
-      secondary_link_text='Not received an email?'
+      secondary_link_text="Didn't receive an email?"
```

Comment:
It's a nit, but can you use the fancy `’` single quote to match the rest of the site’s copy 😎

Reply:
gross. yeah